nilearn/image/tests/test_resampling.py

```diff
@@ -558,6 +530,44 @@ def test_resample_to_img():
     np.testing.assert_almost_equal(downsampled,
                                    result_img.get_data()[:x, :y, :z, ...])
 
+def test_crop():
```

Comment:
Maybe a small comment explaining the logic of this test. It helps future developers.